sc2ts/cli.py

```diff
@@ -229,9 +235,11 @@ def daily_extend(
             num_threads=num_threads,
             show_progress=not no_progress,
         )
-        for ts, date in ts_iter:
-            output = output_prefix + date + ".ts"
-            add_provenance(ts, output)
+        for ts, excluded_samples, date in ts_iter:
+            output_ts = output_prefix + date + ".ts"
+            add_provenance(ts, output_ts)
+            output_excluded_samples = output_prefix + date + ".pickle"
```

Comment:
Maybe add descriptive ".excluded.pickle" or something so we know what they are?

Reply:
I'm going for ".excluded_samples.pickle". A bit long, but pretty clear what it is.